msteams-platform/toolkit/visual-studio-overview.md

```diff
@@ -15,45 +15,37 @@ The Microsoft Teams Toolkit enables you to create custom Teams apps directly wit
+2. Make sure that the **<span>ASP.NET</span> and web development module** has been added to your Visual Studio instance. You can check by following the steps in the [Modify Visual Studio by adding or removing workloads and component](/visualstudio/install/modify-visual-studio?view=vs-2019&preserve-view=true) documentation.
 
 ![visual studio asp.net module](../assets/images/visual-studio-web-dev-module.png)
 
-3. If you would like test your app by deploying it from Visual Studio, you'll need to have IIS (Internet Information Services) installed in your development environment. Visual Studio does not include IIS and it isn't included in the default Windows 10, Windows 8, or Windows 7 configuration; however, you can download the latest version from the [Microsoft download center](https://www.microsoft.com/download/details.aspx?id=48264).
-
-![IIS download page view](../assets/images/iis.png)
-
 ## Install the Teams Toolkit
 
-The Microsoft Teams Toolkit for Visual Studio is available for download from the [Visual Studio Marketplace](https://marketplace.visualstudio.com/items?itemName=TeamsDevApp.vsteamstemplate) or directly from the **Extensions** menu within Visual Studio.
+The Microsoft Teams Toolkit for Visual Studio is available for download from the [Visual Studio Marketplace](https://marketplace.visualstudio.com/items?itemName=msft-vsteamstoolkit.vsteamstoolkit) or directly from the **Extensions** menu within Visual Studio.
 
 ## Using the toolkit
 
 - [Set up a new project](#set-up-a-new-teams-project)
 - [Configure your app](#configure-your-app)
-- [Package your app](#package-your-app)
 - [Run your app in Teams](#install-and-run-your-app-locally)
 - [Validate your app](#validate-your-app)
 - [Publish your app](#publish-your-app-to-teams)
 
 ## Set up a new Teams project
 
-1. Select **Create a new project**.
-1. Choose **Microsoft Teams App** and select **Next**.
-1. You will arrive at the **Configure your new project** screen where you can choose the **Project name**, **Location**, and **Solution name**.
-1. Check the box labeled **Place solution and project in the same directory**.
-1. A pop-up window labeled **Add Capabilities** will allow you to choose one or more capabilities for your project setup.
-1. Select the **Next** button to complete the configuration process.
-1. A pop-up window labeled **Add Capabilities** will allow you to choose the properties for each selected capability.
-1. Select **Finish** and you will  land on the **Microsoft Teams Toolkit** landing page.
+1. Launch Visual Studio 2019.
+2. Select **Create a new project**.
+3. Search for **Microsoft Teams App** and select **Next**.
+4. You will arrive at the **Configure your new project** screen where you can choose the **Project name**, **Location**, and **Solution name**.
+5. Select the **Next** button to choose a name for the app.
```

Comment:
5. Select **Next** to enter a name for the app.

Question: Please clarify the resulting screen post selecting Next. Are you selecting or entering the app name? Please change the process step as per the action.

Reply:
Please rephrase step 5 to "Select Next to enter a name for the app."

Reply:
Please incorporate this suggestion rephrase step 5 to "Select Next to enter a name for the app."